libvirt/tests/src/numa/numa_config_with_auto_placement.py

```diff
@@ -197,23 +197,18 @@ def check_cgget_output(test, cgget_message):
     if not slice_line:
         test.error("'machine-qemu' is not found in 'systemd-cgls cpuset' output")
     slice_line = slice_line.replace('\\', '\\\\')[2:]
+
+    cgtest = libvirt_cgroup.CgroupTest('')
+    is_cgroup2 = True if cgtest.is_cgroup_v2_enabled() else False
```

Comment:
How about changing it as below:
```suggestion
    is_cgroup2 = cgtest.is_cgroup_v2_enabled()
```